lib/libc/math/lib_logf.c

```diff
@@ -57,34 +71,48 @@ float logf(float x)
 {
```

Comment:
how about checking the x?

Reply:
As this is the original code, I will update it in next Pull Request.